### PR TITLE
Add reader for DSCOVR EPIC L1b data in HDF5 format.

### PR DESCRIPTION
**Pull request by simonrp84**
This PR adds a reader for [DSCOVR EPIC](https://epic.gsfc.nasa.gov/) data at level 1B, which is in HDF5 format. Level 1A and level 2+ data are not supported by this reader.

I have also added two composites, `true_color` and `true_color_raw`. But note that the `true_color` composite may not look as expected as DSCOVR is in a very different orbit to most sensors and hence gives an unusual view of the Earth with many high zenith angles.

An example `true_color_raw` image:
true_col_raw

 - [x] Tests added
 - [x] Fully documented

### CODE REVIEW

**Comment by TAlonglong** [comment]
duplicate of #1150 ?

**Comment by simonrp84** [comment]
Aah, I hadn't realised there was an older PR for this. Sorry!
Also, no idea why the docs are failing - looks like it comes from something unconeccted with my PR...

**Comment by pnuu** [comment]
There was a typo in a deprecated compositor docstring that caused RtD builds to error, it was fixed in a PR I merged yesterday.

**Comment by simonrp84** [comment]
I've now refactored this a bit, which will hopefully keep codebeat happy. Thanks to @pnuu for the code suggestions!

File: satpy/readers/epic_l1b_h5.py [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+
+
+class DSCOVREPICL1BH5FileHandler(HDF5FileHandler):
+    """File handler for DSCOVR EPIC L1b data."""
+
+    def __init__(self, filename, filename_info, filetype_info):
+        """Init filehandler."""
+        super(DSCOVREPICL1BH5FileHandler, self).__init__(filename, filename_info, filetype_info)
+
+        self.sensor = 'EPIC'
```
[the remark this answers is not in the document]
Comment (mraspaud) [reply satpy/readers/epic_l1b_h5.py:71]: Agreed

[the remark this answers is not in the document]
Comment (simonrp84) [reply satpy/readers/epic_l1b_h5.py:71]: I copied the format from another reader, can check which later. Shall I change the style of that reader name too?

**Comment by simonrp84** [comment]
Have updated this now to fix the issues raised in review.

File: satpy/readers/epic_l1b_h5.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (pnuu) [reply satpy/readers/epic_l1b_h5.py:71]: The sensor is still in caps here.

**Review by mraspaud — APPROVED** [review]
LGTM! Thanks for adding this reader. I suppose the next step is to add the SRFs to pyspectral?

**Pull request merged**